### PR TITLE
Task100: create allure report 

### PR DESCRIPTION
**Pull request by AlexandarPetsoranski**
In this PR: 

- Allure report structure created 
- Screenshot on test fail method added 
- Tests grouped by categories 
- Added:  
     @Severity
    @AllureId
    @Description
    @DisplayName
    
    Also some methods are marked as different @Steps

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -61,6 +95,24 @@
...
+            <artifactId>allure-junit5</artifactId>
+            <version>2.17.0</version>
+            <scope>test</scope>
+        </dependency>
+
+        <dependency>
+            <groupId>junit</groupId>
+            <artifactId>junit</artifactId>
+            <version>4.13</version>
+        </dependency>
```
Comment (EugenBorisik) [inline pom.xml]: Remove old junit

File: src/main/java/pages/yandex/HomePage.java [hunk 2]
```diff
@@ -1,29 +1,35 @@
...
-        return new LogInPage(driver);
+        return new LogInPage();
+    }
+
+    public boolean isHomePageDisplayed() {
+        return driver.findElement(CREATE_ACCOUNT_BUTTON).isDisplayed() &&
+                driver.findElement(LOG_IN_BUTTON).isDisplayed();
+    }
+
+    public WebElement get_NOT_EXISTING_ELEMENT (){
```
Comment (EugenBorisik) [inline src/main/java/pages/yandex/HomePage.java]: Incorrect method naming

File: allure-results/environment.xml [hunk 3]
```diff
@@ -0,0 +1,14 @@
+<environment>
```
Comment (EugenBorisik) [inline allure-results/environment.xml]: It will be hard to maintain this information manually, find the way to add it programmatically

**Review by EugenBorisik — APPROVED** [review]

File: src/main/java/pages/yandex/HomePage.java (hunk 2 above)
Comment (AlexandarPetsoranski) [reply src/main/java/pages/yandex/HomePage.java]: getNotExistingElement

**Pull request merged**